scripts/HealthCheck.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+/**
+ * Class used for container health check by docker compose
+ */
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        var client = HttpClient.newHttpClient();
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:8080/actuator/health"))
```

Comment:
Same here, could we use `server.port`? https://github.com/UKHomeOffice/hocs-templates/blob/a16384322916d2ffd615ce23c26c8dccb38cf76e/src/main/resources/application.properties#L5

Reply:
This file now uses a command line argument to set the end point to check so it is fully customisable